src/Common/src/CoreLib/System/Globalization/CultureInfo.cs

```diff
@@ -1126,7 +1126,7 @@ internal bool HasInvariantCultureName
         }
 
         /// <summary>
-        /// Helper function both both overloads of GetCachedReadOnlyCulture.  If lcid is 0, we use the name.
+        /// Helper function overloads of GetCachedReadOnlyCulture.  If lcid is 0, we use the name.
```

Comment:
💭 I still don't think the first sentence makes sense, but I would agree that 'both both' wasn't helping things. 😄

Reply:
Yeah, It seems the summary is more talking about `<return>` value stuff, but didn't want to change too much